src/renderer/index.js

```diff
@@ -331,6 +333,18 @@ const App = () => {
                                 } Build Path`}
                             />
                         </FormField>
+                        {buildPath && (
+                            <FormField>
+                                <Checkbox
+                                    id="openBuildPathCheckbox"
+                                    checked={openBuildPath}
+                                    handleChange={createHandleToggle(
+                                        'openBuildPath'
+                                    )}
+                                    label="Open folder after building"
```

Comment:
Only comment is this label...any other ideas on what this should say?

Reply:
Hmm actually I dont know what would be a good description for this. I'm open for suggestions.